flexmeasures/api/v3_0/tests/test_sensors_api.py

```diff
@@ -453,29 +513,33 @@ def test_fetch_sensor_stats(
         response = client.get(
             url_for("SensorAPI:get_stats", id=sensor_id),
         )
-        print("Server responded with:\n%s" % response.json)
```

Comment:
These can usually stay, pytest only prints this in case of an error, and then it is rather useful.